src/sessions/PageManager.ts

```diff
@@ -136,6 +137,7 @@ export class PageManager {
 
     private createNextPage(currentPage: Page, pageId: string) {
         let startTime = Date.now();
+        const timeOnParentPage = startTime - currentPage.start;
```

Comment:
suggestion: `timeOnParentPage` is more concise. Use `timeOnParentPage` instead of `timeSpentOnParentPage`.

Reply:
updated in revision and schemas